gallery/src/styles/CssVariables/CssVariablesPage.tsx

```diff
@@ -320,6 +320,18 @@ const CssVariablesPage = () => {
             <ColorDocRow colorVar="--nx-color-validation-invalid">
               The color used on validatable form fields when they are invalid.
             </ColorDocRow>
+            <ColorDocRow colorVar="--nx-color-alert-info">
```

Comment:
These changes shouldn't be showing up in this diff, something is merged wrong or out of date.  To fix, it should work to merge main into the feature branch and then merge the feature branch into this branch.  And then in the future, avoid merging main directly into this branch.